phpunit.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit
+    backupGlobals               = "false"
+    backupStaticAttributes      = "false"
+    colors                      = "true"
+    convertErrorsToExceptions   = "true"
+    convertNoticesToExceptions  = "true"
+    convertWarningsToExceptions = "true"
+    processIsolation            = "false"
+    stopOnFailure               = "false"
+    bootstrap                   = "vendor/autoload.php"
+>
+    <testsuites>
+        <testsuite name="FINDOLOGIC request library">
```

Comment:
To be consistent with `composer.json` and the repository description, please change this to `Library for FINDOLOGIC API requests`.